core/rewards.lua

```diff
@@ -356,6 +359,39 @@ function Spell:GetStatus()
     return collected and Green(L['known']) or Red(L['missing'])
 end
 
+-------------------------------------------------------------------------------
+------------------------------------ TITLE ------------------------------------
+-------------------------------------------------------------------------------
+
+local Title = Class('Title', Reward, {type = L['title']})
+
+function Title:GetText()
+    local text = self.pattern
+    local titleName, _ = GetTitleName(self.id)
+    local title = strtrim(titleName)
+    text = string.gsub(text, '{title}', title)
+    local player = UnitName('player')
+    text = string.gsub(text, '{player}', player)
+    text = White(text)
+    if self.type then text = text .. ' (' .. self.type .. ')' end
+    if self.note then
+        text = text .. ' (' .. ns.RenderLinks(self.note, true) .. ')'
+    end
+    return text
+end
+
+function Title:IsObtained()
+    local _, _, _, completed, _, _, _, _, _, _, _, _, earnedByMe =
+        GetAchievementInfo(self.criteria)
+    completed = completed and (not ns:GetOpt('use_char_achieves') or earnedByMe)
+    if completed then return true end
+    return false
+end
+
+function Title:GetStatus()
+    return self:IsObtained() and Green(L['known']) or Red(L['missing'])
+end
+
```

Comment:
I think that using **IsTilteKnown(MaskID)** is a better solution, since there are two different achievements for Horde and Alliance for the stable master and with this you only need the MaskID of the title.

[Stable Master at wow.tools](https://wow.tools/dbc/?dbc=chartitles&build=9.2.5.44730#page=1&search=277)

/dump IsTitleKnown(277)
/dump GetTitleName(277)

Reply:
Great suggestion! I implemented IsTitleKnown() in my latest push. Thank you!

Reply:
I'll need to go back and update how SL is checking titles but I fixed the title for [Predator] too.